package.json

```diff
@@ -16,8 +16,9 @@
     "@zxing/browser": "^0.1.1",
     "date-fns": "^2.11.1",
     "gray-matter": "^4.0.2",
-    "next": "latest",
+    "next": "12.1.6",
```

Comment:
Had to set this to a specific version, since the installer for next-themes threw an error. Can be set back to "latest" now.